mojap_metadata/converters/glue_converter/__init__.py

```diff
@@ -491,7 +538,34 @@ def generate_from_meta(
                 database_name, f"msck repair table {database_name}.{metadata.name}"
             )
 
-    def generate_to_meta(self, database: str, table: str) -> Metadata:
+    def generate_to_meta(
+        self,
+        database: str,
+        table: str,
+        glue_table_properties: List[str] = None,
+        update_primary_key: bool = False,
```

Comment:
whats the thinking behind calling this `update_primary_key` vs say `get_primary_key`. From what i understand its not actually updating anything, its a flag to say whether to include the primary_key (if availble in glue-table-properties) in the `Metadata` object returned.

edit: ah i kind of see the reasoning as part of `_get_table_parameters` as that does take an existing `Metadata` object and "update" the primary key (from nothing to potenitally something). But given users will never directly call `_get_table_parameters` and so basically the choice is retrieve the `primary_key` from the glue-table-proprties or not I think the name "update_primary_key" is potentially confusing. Happy to be overruled on this though!

Reply:
My thought process behind using _update_ was because we're updating the primary_key in the metadata but `get_primary_key` makes more sense in `generate_to_meta` since the argument is for users to specify if they want to get the primary_key value from the Glue Data Catalog.

So I've updated it to `get_primary_key` in `generate_to_meta` but left as `update_primary_key` in `_get_table_parameters`. Let me know if you want it updated in `_get_table_parameters` too?

Reply:
Might as well change it for consistency but less important since private :-)

Reply:
Makes sense, have updated it in the private method too, thank you!